roles/gluster_brick/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Add bricks to a volume
+  gluster_volume:
+        state: "{{ gluster_cluster_state | default('present') }}"
```

Comment:
Since it is add-brick there is no other state. So you can hard-code it and not allow anyone to give any other state.